add warning note that NLS eReader Zoomax is not supported (#16628)
nvaccess/nvda: user_docs/en/userGuide.md

* APH Chameleon 20
* Humanware BrailleOne
* NLS eReader

>   * Note that the Zoomax is currently not supported

Reviewer: My understanding is that the Zoomax reader DOES work with NVDA however requires a driver supplied by the NLS, so perhaps we should say:

```suggestion
  * Note that the Zoomax NLS eReader requires a separate driver supplies by NLS

```

Reviewer 2: @Qchristensen It works with an add-on driver currently. In our private discussions, you asked them for an updated version compatible with NVDA 2024.1 a couple months ago, and they have yet to provide one.
Therefore, it only works with a driver that must be obtained somehow, and have its compatibility overridden to use.
On top of that, the driver only partially works (output only, not input).
I'm not objecting to your proposed change, but pointing out some caution.

That said,
```suggestion
  * Note that the Zoomax is currently only supported for reading (not typing),; and to use it you must obtain an add-on from NLS
```

Author: ```suggestion
  * Note that the Zoomax is currently not supported without external drivers
```
